### PR TITLE
Update model docs.

### PR DESCRIPTION
**Pull request by tomasatdatabricks**
## What changes are proposed in this pull request?
Two small doc changes:
1. Add the R function flavor (crate) to the list of built in flavors.
2. Document the new docker build for models.
 ## How is this patch tested?
N/A 
## Release Notes
 N/A
### Is this a user-facing change? 
- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.
 
(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)
 
### What component(s) does this PR affect?
 
- [ ] UI
- [ ] CLI 
- [ ] API 
- [ ] REST-API 
- [ ] Examples 
- [x] Docs
- [ ] Tracking
- [ ] Projects 
- [ ] Artifacts 
- [ ] Models 
- [ ] Scoring 
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:
 
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [x] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: docs/source/models.rst [hunk 1]
```diff
@@ -147,6 +147,15 @@ The :py:mod:`mlflow.pyfunc` module defines functions for saving and loading MLfl
 For more information, see the :ref:`custom Python models documentation <custom-python-models>`
 and the :mod:`mlflow.pyfunc` documentation.
 
+R Function (``crate``)
+^^^^^^^^^^^^^^^^^^^^^^
+
+The ``crate`` model flavor defines a generic model format for representing aribtrary R predict
```
Comment (sueann) [inline docs/source/models.rst]: predict -> prediction

File: docs/source/models.rst [hunk 2]
```diff
@@ -147,6 +147,15 @@ The :py:mod:`mlflow.pyfunc` module defines functions for saving and loading MLfl
...
+function as Mlflow model. The predict function is expected to take a dataframe on the  input and
```
Comment (sueann) [inline docs/source/models.rst]: predict -> prediction, or ``predict`` ?

Comment (sueann) [inline docs/source/models.rst]: functions, models

Comment (sueann) [reply docs/source/models.rst]: on the input -> as input

File: docs/source/models.rst [hunk 3]
```diff
@@ -147,6 +147,15 @@ The :py:mod:`mlflow.pyfunc` module defines functions for saving and loading MLfl
...
+produce a dataframe or a vector or a list with the predictions on the output.
```
Comment (sueann) [inline docs/source/models.rst]: a dataframe, a vector, or a list

Comment (sueann) [reply docs/source/models.rst]: on the output -> as the output

File: docs/source/models.rst [hunk 4]
```diff
@@ -171,14 +180,15 @@ For more information, see :py:mod:`mlflow.h2o`.
...
-:py:func:`log_model() <mlflow.keras.log_model>` functions that you can use to save Keras models
-in MLflow Model format. These functions serialize Keras models as HDF5 files using the Keras
-library's built-in model persistence functions. MLflow Models produced by these functions
-also contain the ``python_function`` flavor, allowing them to be interpreted as generic
-Python functions for inference via :py:func:`mlflow.pyfunc.load_pyfunc()`. Finally, you can use
-the :py:func:`mlflow.keras.load_model()` function to load MLflow Models with the
+The ``keras`` model flavor enables logging and loading Keras models. It is available in both python
+and R clients. The :py:mod:`mlflow.keras` module defines :py:func:`save_model()<mlflow.keras.save_model>`
+and :py:func:`log_model() <mlflow.keras.log_model>` functions that you can use to save Keras models
+in MLflow Model format in python. Similarly, in R, you can save or log the model using
```
Comment (sueann) [inline docs/source/models.rst]: python -> Python

File: docs/source/models.rst [hunk 5]
```diff
@@ -171,14 +180,15 @@ For more information, see :py:mod:`mlflow.h2o`.
...
+:rlang:func:`mlflow_save_model` and :rlang:func:`mlflow_log_model. These functions serialize Keras
+models as HDF5 files using the Keras library's built-in model persistence functions. MLflow Models
+produced by these functions also contain the ``python_function`` flavor, allowing them to be interpreted
+as generic Python functions for inference via :py:func:`mlflow.pyfunc.load_pyfunc()`. Finally, you
+can use the :py:func:`mlflow.keras.load_model()` function to load MLflow Models with the
```
Comment (sueann) [inline docs/source/models.rst]: is there an equivalent in R we should mention?

File: docs/source/models.rst [hunk 6]
```diff
@@ -457,11 +467,16 @@ Not all deployment methods are available for all model flavors.
...
 
-Deploy MLflow models locally
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-MLflow can deploy models locally as local REST API endpoints or to directly score files.
-You deploy MLflow model locally using the CLI interface to the :py:mod:`mlflow.models` module.
-The local REST API server accepts the following data formats as inputs:
+Deploy MLflow models
+^^^^^^^^^^^^^^^^^^^^
+MLflow can deploy models locally as local REST API endpoints or to directly score files. In addition,
+MLflow can package models as self contained docker images with the REST api endpoint. The image can
```
Comment (sueann) [inline docs/source/models.rst]: api -> API

**Review by sueann — COMMENTED** [review]
some language nits - thanks!

File: docs/source/models.rst (hunk 2 above)
Comment (tomasatdatabricks) [reply docs/source/models.rst]: I guess prediction makes more sense. thanks.

File: docs/source/models.rst (hunk 5 above)
Comment (tomasatdatabricks) [reply docs/source/models.rst]: Yes, good point.

File: docs/source/models.rst [hunk 7]
```diff
@@ -180,16 +180,17 @@ For more information, see :py:mod:`mlflow.h2o`.
...
-:rlang:func:`mlflow_save_model` and :rlang:func:`mlflow_log_model. These functions serialize Keras
+in MLflow Model format in Python. Similarly, in R, you can save or log the model using
+`mlflow_save_model <#mlflow-save-model>`__ and `mlflow_log_model <#mlflow-log-model>`__. These functions serialize Keras
 models as HDF5 files using the Keras library's built-in model persistence functions. MLflow Models
 produced by these functions also contain the ``python_function`` flavor, allowing them to be interpreted
 as generic Python functions for inference via :py:func:`mlflow.pyfunc.load_pyfunc()`. Finally, you
-can use the :py:func:`mlflow.keras.load_model()` function to load MLflow Models with the
-``keras`` flavor as `Keras Model objects <https://keras.io/models/about-keras-models/>`_.
+can use the :py:func:`mlflow.keras.load_model()` in Python or `mlflow_load_model <#mlflow-load-model>`__
+in R function to load MLflow Models with the ``keras`` flavor as
```
Comment (sueann) [inline docs/source/models.rst]: "function" should come after ":py:func:`mlflow.keras.load_model()`"

File: docs/source/models.rst [hunk 8]
```diff
@@ -469,9 +470,9 @@ Not all deployment methods are available for all model flavors.
 
 Deploy MLflow models
 ^^^^^^^^^^^^^^^^^^^^
-MLflow can deploy models locally as local REST API endpoints or to directly score files. MLflow can
-also package REST api endpoitn serving the model as a self contained docker image. The image can be
-used to safely deploy the model to various environments such as Kubernetes.
+MLflow can deploy models locally as local REST API endpoints or to directly score files. In addition,
+MLflow can package models as self contained docker images with the REST API endpoint. The image can
```
Comment (sueann) [inline docs/source/models.rst]: docker -> Docker

**Review by sueann — APPROVED** [review]
just a few nits - feel free to merge after they are fixed. thanks!

**Pull request merged**